NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+export("#examples")
```

Comment:
@chariga98 do you know where this has come from? I can't work it out, but I don't think we want it! I'll manually remove it if need be.

Reply:
I'll remove this now, so I can approve and merge. Great work!